### PR TITLE
[perftests] Find ninja with xcrun when not in llbuild-test-tools

### PR DESCRIPTION
**Pull request by dmbryson**

### CODE REVIEW

**Comment by dmbryson** [comment]
@swift-ci please smoke test

File: perftests/Xcode/PerfTests/NinjaPerfTests.mm [hunk 1]
```diff
@@ -30,6 +30,15 @@ static void ExecuteShellCommand(const char *String) {
 
 @implementation NinjaPerfTests
 
++ (NSString*) findNinja {
+    NSString* overrideNinjaPath = [@(SRCROOT)
+                                   stringByAppendingPathComponent:@"llbuild-test-tools/utils/Xcode/ninja"];
```
Comment (jakepetroules) [inline perftests/Xcode/PerfTests/NinjaPerfTests.mm]: Is this even really needed?

Comment (dmbryson) [reply perftests/Xcode/PerfTests/NinjaPerfTests.mm]: I expect that the perf test bots are set up this way.  My goal here was to not change the existing behavior for those that have it set up as such, while adding a sane default.  You could conceivably want to override what version of ninja you want to compare against.

**Comment by dmbryson** [comment]
Ping

**Pull request merged**